### ISSUE TITLE
Strange formatting of help drawer on blank section

### ISSUE THREAD

**Issue by jwirfs-brock** [denverfoundation/storybase#762]
I created a new Photovoice story and didn't edit anything. I clicked on the middle "Untitled Section" and opened the help drawer.

Here's a screenshot of what I saw:
![screen shot 2013-05-13 at 10 10 57 am](https://f.cloud.github.com/assets/1020518/495995/19b89012-bbd7-11e2-9733-cef2004bd84b.png)

(1) I couldn't scroll to see the full help drawer text.
(2) The help drawer appears to start well below the top of the screen. I think this might have been a fix for something else (maybe the browser and asset messages that pop up across the top?), but it looks really strange.
(3) In the asset containers, the asset type icons are overlapping the "+" icon.

**Comment by jwirfs-brock** [comment]
Hmmm, note that in this screen shot, from a sandbox story, the help drawer opens as expected:
![screen shot 2013-05-13 at 10 24 38 am](https://f.cloud.github.com/assets/1020518/496067/f114ebcc-bbd8-11e2-9263-e52eac74b81c.png)

Both were done in Chrome.

**Comment by jwirfs-brock** [comment]
I'm also seeing this on the Story Information section in the Data Story layout:
http://floodlightproject.org/en/build/2196782de28b42808e0471485a0f8720/

(Using Firefox 21.0 on a Mac.)

**Comment by ghing** [comment]
@jwirfs-brock I wasn't able to replicate this, either in a recent version of Chrome or Firefox. Do you always see this behavior, or is it intermittent. Figuring out how to consistently reproduce this behavior would be a big help.

**Comment by jwirfs-brock** [comment]
@ghing I am seeing this intermittently. Just now, I tried created a couple of new stories. I saw this bug in the Data Story, but did not see it in the new PhotoVoice or Sandbox stories I created.

Any tips on how to get more detail about the bug if it's happening intermittently? I already put my browser and OS info on the GitHub issue... would it help to get the stuff logged in the console, perhaps?

**Comment by ghing** [comment]
I don't think this issue will fire an error, but having the console enabled (and switching to it after the condition appears) would be good to double check.

What might be more helpful, would be to copy and paste the HTML from Firebug or the Firefox/Chrome development tools' HTML inspector.  That way we can see if the offending space is caused by an elements style being set by JavaScript, or a particular class, which will make it easier to find the place in the code where it's happening.  Note that you'll have to use the developer tools because a simple view source won't show changes to the DOM made dynamically, after the initial load.

**Comment by jwirfs-brock** [comment]
Ok, here is the HTML inspector:

  <!DOCTYPE html>
      

```
  <!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en"> <![endif]-->

  <!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en"> <![endif]-->

  <!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en"> <![endif]-->

  <!-- Consider adding an manifest.appcache: h5bp.com/d/Offline -->

  <!--[if gt IE 8]><!-->
<html class=" js history filereader" lang="en">
  <!--<![endif]-->
<head> … </head><body class="story-builder wysihtml5-supported" style="overflow-x: auto;"><div id="builder" class=""><header id="header"><div id="title-bar"><div id="title-bar-contents"><h1 class="title"> … </h1><ul id="tools" class="nav"> … </ul></div></div><div id="workflow-bar"> … </div><div id="subnav-bar"> … </div><div id="alerts"></div></header><div id="app" style="margin-top: 348px;"><div class="builder"></div></div><div id="drawer-container" style="margin-top: 348px;"> … </div></div><script> … </script>
  <!-- Grab Google CDN's jQuery, with a protocol relative URL; fall back to local if offline -->
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script><script> … </script><script src="/static/js/plugins.js" type="text/javascript"></script><script src="/jsi18n/" type="text/javascript"></script><script src="/static/js/libs/underscore-min.js"></script><script src="/static/js/libs/json2.js"></script><script src="/static/js/libs/backbone-min.js"></script><script src="/static/js/libs/handlebars-1.0.0.beta.6.js"></script><script src="/static/js/libs/backbone-forms/backbone-forms.min.js"></script>
  <!-- TODO: Finalize editor matching views -->
<script src="/static/js/libs/wysihtml5/advanced.js"></script><script src="/static/js/libs/wysihtml5/wysihtml5-0.3.0.min.js"></script><script src="/static/js/libs/jquery-ui-1.8.22.custom.min.js"></script><script src="/static/js/libs/jquery.jeditable.mini.js"></script><script src="/static/js/libs/select2/select2.min.js"></script><script src="/static/js/libs/leaflet/leaflet.js"></script><script src="/static/js/libs/jquery.cookie.js"></script><script src="/static/js/libs/guiders/guiders-1.2.8.js"></script><script src="/static/js/libs/tooltipster-1.2/js/jquery.tooltipster.min.js"></script><script src="/static/js/builder/jquery.graphicselect.js"></script><script src="/static/js/jquery.storybaseshare.js"></script><script src="/static/CACHE/js/004435cfcae9.js" type="text/javascript"></script><script> … </script><script type="text/javascript"> … </script></body></html>
```

And here's the CSS inspector for the "help" element:

element {
}
- {
  margin: 0px;
  padding: 0px;
  outline: 0px none;
  }
  body, button, input, select, textarea {
  font-family: Arial,sans-serif;
  font-size: 14px;
  line-height: 1.4em;
  color: rgb(68, 68, 68);
  }
  html {
  font-size: 100%;
  }

**Comment by jwirfs-brock** [comment]
@ghing it looks like some of the HTML was truncated. Let me know if you want to do a screen share to get a better view.

**Comment by mgrayinnovation** [comment]
Moved to index of issues for reference during Pivot: https://docs.google.com/document/d/1dmlOEQJwYx0MddonRLmtN_zAitJ-lJyPm5W272nl6U8/edit
